src/router/index.js

```diff
@@ -78,6 +91,11 @@ export default new Router({
       component: EditCluster
     },
     {
+      path: '/projects/:project_id/subscriptions/:subscription_id/edit',
+      name: 'EditSubscription',
+      component: EditSubscription
```

Comment:
This answers my previous question (in the other review), so yes we have support to edit them. Maybe I didn't see the test case for the subscription edit.

Reply:
@taliaga Though this is editing the subscription entry in the MasterMind API database, not updating (patching) the sub inside a context broker yet. That will need to be implemented later.

Reply:
Oh, I see. Let's create an issue so as not to risk living with the inconsistency. I'm not sure if you prefer an issue in the API repo or if you're managing this in JIRA, so I'll leave it up to you :)